tests/class/SC_Product/SC_Product_listsTest.php

```diff
@@ -82,7 +82,7 @@ public function testlists_商品一覧取得()
             ),
         );
 
-        $this->actual = $this->objProducts->lists($this->objQuery);
+        $this->actual = $this->objProducts->lists($this->objQuery, array(1001, 1002));
```

Comment:
通常、 SC_Product::lists() を使用する場合は、 SC_Query::setWhere() で `$where` 及び `$arrWhereVal` を設定します
```suggestion
        $this->objQuery->setWhere('product_id IN (?, ?)', array(1001, 1002));
        $this->actual = $this->objProducts->lists($this->objQuery);
```

Reply:
ありがとうございます。
既存2.13の改修中にハマった箇所なので明確な意図が書いてあると後々助かる方が増える気がします。
🙇